### PR TITLE
Update swagger

### PR DESCRIPTION
**Pull request by abourne1**
Summary:
 - Update copy in swagger documentation to be more user friendly using what Devon wrote on hackMD https://hackmd.io/OZk7k5_RT_aTaI1uVIRiVQ?both
 - Fix link to sample bulk request CSV

@kimpenguin @devowhippit before I land/deploy can one of you take a pass at the copy in this diff and tell me how it looks when rendered in the browser?

### CODE REVIEW

**Review by devowhippit — APPROVED** [review]

File: dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml [hunk 1]
```diff
@@ -1081,19 +1084,19 @@ paths:
                               example: null
     options:
       tags:
-        - Eligibility Screening Endpoints
+        - Eligibility
       description: >
-        The /eligibilityPrograms OPTIONS method returns a list of supported HTTP methods, a readable schema and an example request payload.
+        The OPTIONS method will return the HTTP methods that the server supports for the specified URL along with a readable schema and an example of the request payload.
```
Comment (kimpenguin) [inline dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml]: Did we still want to keep the options method?

File: dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml [hunk 2]
```diff
@@ -22,9 +22,11 @@ paths:
   /authToken:
     post:
       tags:
-        - Account Management Endpoints
+        - Authentication
       description: >
-        The /authToken endpoint generates new authentication tokens (they expire every hour) and can also reset your password if the "newPassword" field is present.
+        The /authToken endpoint is used to set your initial account password and retrieve an auth token for making additional requests to the eligibility endpoints. 
+        See “Set Password and Token Based Authentication” for more details on setting your initial account password. When used to retrieve a token the response of this 
+        endpoint must be provided in the request header of the eligibility endpoints to make an authenticated request.
```
Comment (kimpenguin) [inline dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml]: Light edits: This endpoint is used to set your initial account password and retrieve an authentication token to make requests to the **Eligibility** endpoints. 
        See <a href="/set-password-and-token-based-authentication">Set Password and Token Based Authentication</a> for more details on setting your initial account password. 
        On successful response, the retrieved token must be provided in the request header of the **Eligibility** endpoints to make an authenticated request.

File: dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml [hunk 3]
```diff
@@ -219,9 +221,10 @@ paths:
   /forgotPassword:
     post:
       tags:
-        - Account Management Endpoints
+        - Authentication
       description: >
-        The /forgotPassword endpoint triggers the password reset flow. POSTing to this endpoint will cause the API to send an email with password reset instructions.
+        This endpoint is used to trigger a forgotten password email to be sent to the email associated with a user’s account. 
+        The email will include a verification code the user can use to reset their password with the /confirmPassword endpoint.
```
Comment (kimpenguin) [inline dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml]: Light edits:
This endpoint is used to trigger a forgotten password email that is sent to the email associated with a user’s account. 
        The email includes a verification code that the user will use to reset their password with the /confirmPassword endpoint.

File: dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml [hunk 4]
```diff
@@ -599,10 +602,10 @@ paths:
   /eligibilityPrograms:
     post:
       tags:
-        - Eligibility Screening Endpoints
+        - Eligibility
       description: >
-        The /eligibilityPrograms endpoint returns a list of public benefits that an individual may be eligible for. 
-        Possible request fields are specified in the "Request body" section below.
+        This is the endpoint for making single requests containing household composition data to retrieve response that contains program names and codes. 
+        See “Making a Request” for more details on household composition data and the response codes.
```
Comment (kimpenguin) [inline dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml]: This endpoint is used to make single requests to retrieve a response containing potentially eligible program names and codes based on household composition data provided by the user.
        See <a href="/making-a-request">Making a Request</a> for more details on household composition data and the response codes.

File: dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml [hunk 5]
```diff
@@ -1081,19 +1084,19 @@ paths:
...
   /bulkSubmission/import:
     post:
       tags:
-        - Eligibility Screening Endpoints
+        - Eligibility
       description: >-
-        The /bulkSubmission/import endpoint returns a list of public benefits in a structured CSV format that a group of people may be eligible for. 
-        An example input CSV is available in our Postman Collection.
+        This endpoint accepts a structured CSV of multiple household compositions allowing users to query for eligibility in bulk. The response back will be another CSV with the program names and codes. 
+        See “Making a Request” for more details on household composition data and the response codes. Click <a href="/resources/sample_bulk_submission_import.csv">here</a> to download a sample request CSV.
```
Comment (kimpenguin) [inline dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml]: This endpoint accepts a structured CSV of multiple household compositions allowing users to query for potential eligibility in bulk. The response will be a CSV containing the potentially eligible program names and codes. 
        See <a href="/making-a-request">Making a Request</a> for more details on household composition data and the response codes. Click <a href="/resources/sample_bulk_submission_import.csv">here</a> to download a sample request CSV.

**Review by kimpenguin — APPROVED** [review]
Looks good, man! Added some edits and links to documentation pages.

File: dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml (hunk 2 above)
Comment (abourne1) [reply dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml]: agreed!

File: dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml (hunk 1 above)
Comment (abourne1) [reply dist/68d422386f1c9b95dab97295f2644aa1687647a4/resources/access_eligibility_api.yaml]: idk, it doesn't hurt? But maybe it's confusing and hopefully our documentation is good enough that we don't need an OPTIONS method : p

I'll remove the documentation for it for now.

**Comment by abourne1** [comment]
All your edits look good to me @kimpenguin! I'll make them before landing/deploying.

**Pull request merged**